docs/release-notes/self-hosted/self-hosted-v13.0.0.md

```diff
@@ -0,0 +1,160 @@
+-   https://codacy.atlassian.net/browse/TS-456
+-   https://codacy.atlassian.net/browse/TS-374
+-   https://codacy.atlassian.net/browse/TS-336
+-   https://codacy.atlassian.net/browse/TS-110
+-   https://codacy.atlassian.net/browse/CY-7084
+-->
+
+## Upgrading Codacy Self-hosted
+
+Follow the steps below to upgrade to Codacy Self-hosted v13.0.0:
+
+1.  Check the [release notes for all Codacy Self-hosted versions](../index.md#self-hosted) **between your current version and the most recent version** for breaking changes and follow the instructions provided <span class="skip-vale">carefully</span>.
+
+1.  Follow the instructions to [upgrade your Codacy Self-hosted instance](https://docs.codacy.com/v13.0/chart/maintenance/upgrade/).
+
+1.  Update your Codacy command-line tools to the following versions:<!--TODO Update CLI tool versions-->
+
+    -   [Codacy Analysis CLI MAJOR.MINOR.PATCH](https://github.com/codacy/codacy-analysis-cli/releases/tag/MAJOR.MINOR.PATCH)
+    -   [Codacy Coverage Reporter MAJOR.MINOR.PATCH](https://github.com/codacy/codacy-coverage-reporter/releases/tag/MAJOR.MINOR.PATCH)
+
+## Product enhancements
+
+-   Codacy now displays the coverage variation metric with a precision of two decimal places on the [Pull request](https://docs.codacy.com/v13.0/repositories/pull-requests/), [Commit](https://docs.codacy.com/v13.0/repositories/commits/), and [Files](https://docs.codacy.com/v13.0/repositories/files/) page, and you can [define quality gates](https://docs.codacy.com/v13.0/repositories-configure/adjusting-quality-settings/#gates) with a coverage variation threshold using the same precision.
+
+    The increased precision of the metric reflects the code coverage changes better by reducing issues with rounding errors. ![Coverage variation on the Pull request quality overview](../images/io-92.png) (IO-92)
+
+## Bug fixes
+
+-   Pylint now supports Python 3.11 syntax. (TS-467)
+-   Added support for the ESLint plugin [<span class="skip-vale">vue-scoped-css</span>](https://www.npmjs.com/package/eslint-plugin-vue-scoped-css). (TS-356)
+-   Added support for the ESLint plugin [<span class="skip-vale">eslint-plugin-rxjs</span>](https://www.npmjs.com/package/eslint-plugin-rxjs). (TS-346)
+-   Added support for the ESLint plugins [<span class="skip-vale">eslint-plugin-markdown</span>](https://www.npmjs.com/package/eslint-plugin-markdown) and [<span class="skip-vale">eslint-plugin-yml</span>](https://www.npmjs.com/package/eslint-plugin-yml). (TS-121)
+-   Fixed an incorrect popup being shown when loading the repository Code patterns page. (PLUTO-717)
+
+## Tool versions
+
+This version of Codacy Self-hosted includes the tool versions below. The tools that were updated on this version are highlighted in bold:
+
+-   Ameba 1.4.3
+-   **[Bandit 1.7.5](https://github.com/PyCQA/bandit/releases/tag/1.7.5) (updated from 1.7.0)**
+-   Brakeman 4.3.1
+-   **bundler-audit (deprecated) 0.9.1 (updated from 0.6.1)**
```

Comment:
@stefanvacareanu7 What's the correct way to present the deprecated tools? Using the old or the new version? 🤔 

Using the old version:
```suggestion
-   **bundler-audit (deprecated) 0.6.1**
```

Reply:
Or using the new version:
```suggestion
-   **bundler-audit (deprecated) 0.9.1**
```

Reply:
@nicklem clarified to me that we updated some of the tools we are deprecating, so I updated it accordingly (keeping the new and old versions).